packages/ui/ButtonGroupSimple/index.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import { ReactNode } from "react";
+import styled from "styled-components";
+import { device } from "web/lib/media-queries";
+
+type InputProps = {
+  component: ReactNode;
+  id: string;
+  checked: boolean;
+};
+
+type GroupProps = {
+  options: InputProps[];
+  selectedOption: string;
+  onOptionSelect: (id: string) => void;
+} & React.HTMLProps<HTMLDivElement>;
+
+const ButtonRadioGroupStyled = styled.div`
+  display: flex;
+  gap: 7px;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  height: 100%;
+
+  label {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    padding: 4px 2px;
+    position: relative;
+    text-align: center;
+    border-bottom: 2px solid transparent;
+    color: #7e7e7e;
+
+    ${device.tablet} {
+      padding: 8px 12px;
+    }
+  }
+
+  label:has(input:checked),
+  label:has(input:focus) {
+    border-bottom: 2px solid ${({ theme }) => theme.main};
+    z-index: 1;
+    color: ${({ theme }) => theme.main};
+    font-weight: 600;
+  }
+
+  //hide input without losing accessibility
+  input {
+    height: 0;
+    width: 0;
+    opacity: 0;
+    position: absolute;
+  }
+
+  //label on hover
+  label:hover {
+    border-bottom: 2px solid ${({ theme }) => theme.main};
+    z-index: 1;
+  }
+`;
+
+export const ButtonGroupSimple = ({
+  options,
+  selectedOption,
+  onOptionSelect,
+}: GroupProps) => (
+  <ButtonRadioGroupStyled>
```

Comment:
I'm thinking of adding some arias here
```suggestion
  <ButtonRadioGroupStyled role="radiogroup" aria-orientation="vertical">
```